app/models/lookup_value.rb

```diff
@@ -45,6 +45,7 @@ def name
   end
 
   def value_before_type_cast
+    return read_attribute(:value) if errors.present? && errors[:value].present?
```

Comment:
if errors is blank, won't calling errors[:value] throw a NoMethodError :[] for nil?